examples/advanced/defaults_list_interpolation/my_app.py

```diff
@@ -4,7 +4,7 @@
 import hydra
 
 
-@hydra.main(config_path="conf", config_name="config")
+@hydra.main(version_base=None, config_path="conf", config_name="config")
```

Comment:
Thank you! overall it looks great!
My question is do we need to show `version_base=None` in all the examples? For new users starting afresh they probably won't need to know this till later (there is fair amount to ramp up on when first starting Hydra, so IMO less is better). 
For existing users, they are likely to change this version_base at upgrade time and we will include the pointer to version_base in the upgrade guide.

Reply:
Note that an unspecified version_base currently gives a warning.
Also I was removing settings of config_path=None as that's implied with version_base=None.
Also I didn't add version_base params to most of the intro docs for the reasons you state.
I'll push an update with docs, to hopefully clarify this more.
Happy to adjust after that